01.fizzbuzz/FizzBuzz.rb

```diff
@@ -0,0 +1,13 @@
+21.times do |x|
+    next if x==0
+    
+    if x%5==0 && x%3==0 then
```

Comment:
```suggestion
    if x % 5 == 0 && x % 3 == 0
```

演算子の前後はスペースを入れる、thenは使わない、が一般的です。